components/mixins/account/account-details.schema.json

```diff
@@ -0,0 +1,103 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/mixin/account-details",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Account Details",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/class/account"],
+  "description": "An XDM Business Account.",
+  "definitions": {
+    "account-details": {
+      "properties": {
+        "xdm:accountComponents": {
```

Comment:
If this is an array of b2b-account objects, then why not have the "items" simply be a $ref to that datatype? Is there a reason to duplicate those things here?

Reply:
I also thought to refer that initially but I believe, there are still few fields which are not included in the array that exist in the "b2b-account" data type. Let me double check again.

Reply:
I think, we can refer that data type in the mixin. I just checked. Thanks for pointing this out. I will make the change.